### ISSUE TITLE
Problem with dropout

### ISSUE THREAD

**Issue by truongdo** [truongdo/etra#1]
There might be a bug with dropout implementation that cause the training loss become very large.
